site/docs/v1/tech/shared/_scim-limits.adoc

```diff
@@ -6,3 +6,5 @@ The current SCIM implementation does not support:
 * Query operations
 
 All of these operations are available via link:/docs/v1/tech/apis/[FusionAuth APIs], but are not currently supported via SCIM compatible endpoints. If this functionality is required, please file a https://github.com/fusionauth/fusionauth-issues/issues[GitHub issue] explaining your use case.
+
+While it is not a limitation, it is recommended to configure at most one SCIM server per tenant.
```

Comment:
I mean, I don't think there'd be any issue with using more than one because each SCIM interaction specifies (via the client credentials grant) a specific client and server. So you could run more than one SCIM server off of the same tenant, if you wanted to expose different permissions, primarily. All the tenant config would be the same.

Maybe I misunderstand things; if there is complexity I'm missing, please do expand on this.

Reply:
he issue is that you have more than one Server creating users. So unless you coordinate it - or know for certain they do not overlap, you'll have problems.

Same concept of allowing IdPs to use Link by Username in a sense. It is ok if you use two separate IdPs that you know have usernames that never collide. So it is buyer beware.

Use case: Does SCIM Server B expect that this user should exist already?
SCIM Server A -> Create user Bob -> Success.
SCIM Server B -> Create user Bob -> Fail.

Use case: Does SCIM Server A know that SCIM Server B just stomped on their changes?
SCIM Server A -> Update user Bob -> Success.
SCIM Server B -> Update user Bob -> Success.

Lots of edge cases ensue..  so we allowed it so that we can enable some more advanced use cases, but it could be dicey if you don't know what you are doing.